www/obs/socket.io.js

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Socket.IO v2.1.1
```

Comment:
We can probably use this for sync? Nothing to change right now but clean up in the future. cc @bogas04

Reply:
Yeah we can use it, I dynamically load socket.io lib from our server as that takes care of connection and everything. We can definitely use the library and connect manually